### PR TITLE
Add security label for vulnerability Alerts

### PR DESCRIPTION
**Pull request by taxintt**
https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -32,4 +33,4 @@
       ]
     }
   ]
-}
+}
```
Comment (ikesyo) [inline default.json:36]: I want to preserve final new lines for consistency.

File: vulnerabilityAlerts.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "vulnerabilityAlerts": {
+        "labels": [
+            "security"
+        ]
+    }
+}
```
Comment (ikesyo) [inline vulnerabilityAlerts.json:7]: Same as above.

File: vulnerabilityAlerts.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "vulnerabilityAlerts": {
+        "labels": [
+            "security"
+        ]
+    }
```
Comment (ikesyo) [inline vulnerabilityAlerts.json:6]: This setting is exemplified in the document.

image

**Review by ikesyo — COMMENTED** [review]
Could you please revert final new lines changes? Otherwise looks good 👍

File: default.json (hunk 1 above)
Comment (taxintt) [reply default.json]: Added final line in the file!
[Add final line in json files](https://github.com/hatena/renovate-config/pull/81/commits/fe4814defd7901a522e88fe3ec1ab4bc856eccfa)

File: vulnerabilityAlerts.json (hunk 2 above)
Comment (taxintt) [reply vulnerabilityAlerts.json]: Also added final line
[Add final line in json files](https://github.com/hatena/renovate-config/pull/81/commits/fe4814defd7901a522e88fe3ec1ab4bc856eccfa)

**Review by ikesyo — APPROVED** [review]
👍

**Pull request merged**